cmd/create/network/cmd.go

```diff
@@ -0,0 +1,141 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/network"
+	helper "github.com/openshift/rosa/pkg/network"
+	"github.com/openshift/rosa/pkg/ocm"
+	opts "github.com/openshift/rosa/pkg/options/network"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+func NewNetworkCommand() *cobra.Command {
+	cmd, options := opts.BuildNetworkCommandWithOptions()
+	cmd.Run = rosa.DefaultRunner(rosa.RuntimeWithOCMAndAWS(), NetworkRunner(options))
+	interactive.AddModeFlag(cmd)
+
+	cmd.SetHelpFunc(func(cmd *cobra.Command, args []string) {
+		templateDir := "cmd/create/network/templates"
+		err := filepath.WalkDir(templateDir, func(path string, d fs.DirEntry, err error) error {
+			if err != nil {
+				return err
+			}
+			if !d.IsDir() && strings.HasSuffix(d.Name(), ".yaml") {
+				templateBody, err := os.ReadFile(path)
+				if err != nil {
+					fmt.Println(err)
+					return nil
+				}
+
+				var templateMap map[string]interface{}
+				err = yaml.Unmarshal(templateBody, &templateMap)
+				if err != nil {
+					fmt.Println(err)
+					return nil
+				}
+
+				parameters, ok := templateMap["Parameters"].(map[string]interface{})
+				if !ok {
+					fmt.Printf("No parameters found in the CloudFormation template %s\n", d.Name())
+					return nil
+				}
+
+				fmt.Printf("Available parameters in %s/%s:\n", filepath.Base(filepath.Dir(path)), d.Name())
+				for paramName := range parameters {
+					fmt.Printf("  %s\n", paramName)
```

Comment:
Just noticed youre using print in some spots, is this intended? I think we normally use the logger

Reply:
the use of fmt is intentional otherwise the help command output would look strange if we were to use the logger